### PR TITLE
#106 login page redesigns

### LINKED ISSUE #106: Better first-time setup page
Redesign is needed for the first time setup page, and the "setup new core" page.

More information should also be provided to guide the user
image

### PR DESCRIPTION
**Pull request by Arcslogger**
+ smol tweak of large buttons
closes #106

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -29376,4 +29376,4 @@
       }
     }
   }
-}
\ No newline at end of file
+}
```
Comment (Arcslogger) [inline package-lock.json:29379]: extra line is from prettier

**Review by Ynng — CHANGES_REQUESTED** [review]
Would it be possible have the background centered?
As in, as I shrink the width of the page, the entire page goes black.

If we center the image in the center, maybe we can get some of the colors to show up too?

**Comment by Ynng** [comment]
And some minor issue with redirect.

**Review by Ynng — APPROVED** [review]

**Review by Ynng — CHANGES_REQUESTED** [review]
Actually a few more things: 
- "mixed/insecure content" is not the only network issue that could happen. We should probably link to our wiki instead
- The network settings warning should probably be on the CoreConnect page too, since that's likely when the problems are going to manifest themselves.

**Comment by Ynng** [comment]
`You may need to adjust your network and browser settings. Learn more here`

**Review by Ynng — APPROVED** [review]

**Pull request merged**